libs/components/phone-field/src/lib/modules/phone-field/phone-field.component.spec.ts

```diff
@@ -147,22 +147,29 @@ describe('Phone Field Component', () => {
       | PhoneFieldReactiveTestComponent
       | PhoneFieldInputBoxTestComponent
     >,
+    programmaticIso2 = '',
   ): void {
-    const countryInput = getCountrySearchToggleButton(compFixture);
-    countryInput.click();
-    detectChangesAndTick(compFixture);
+    if (programmaticIso2)
+      compFixture.componentInstance.selectedCountry = {
```

Comment:
Mind wrapping the `if` block in curly brackets?

Reply:
I'll be honest - not sure how I did that because I never write my if-statements like that. I have fixed it.